src/hotspot/share/gc/serial/serialHeap.hpp

```diff
@@ -363,6 +363,9 @@ class SerialHeap : public CollectedHeap {
   void oop_since_save_marks_iterate(OopClosureType1* cur,
```

Comment:
This declaration can go now?

Reply:
Yes, the test `test-tier1_gc` passed locally.

Reply:
Removed just now.